projet/users/views.py

```diff
@@ -1,3 +1,68 @@
 from django.shortcuts import render
+from rest_framework import generics, permissions
+from rest_framework.views import APIView
+from rest_framework.authtoken.views import ObtainAuthToken
+from rest_framework.response import Response
+import json
+from rest_framework.decorators import api_view
+from rest_framework import mixins, generics
+from django.http import HttpResponse, JsonResponse
+from rest_framework.permissions import IsAuthenticated
+from rest_framework import status
+from .models import User
+from rest_framework.authtoken.models import Token
+from django.contrib.auth import authenticate
+from django.contrib.auth.hashers import check_password
+from .serializers import UserSerializer, UserValidateSerializer
+
 
 # Create your views here.
+class UserCreate(APIView):
+    def post(self, request, format="json"):
+        serializer = UserSerializer(data=request.data)
+        if serializer.is_valid():
+            user = serializer.save()
```

Comment:
I'll like to write more tests for this particular view since it handles something as important as authentication.